Tasks/js-targets.js

```diff
@@ -1,11 +1,42 @@
+const fs = require('fs');
```

Comment:
The mentioned linter warnings:
![image](https://user-images.githubusercontent.com/1976647/128041394-ef5fc67b-f24b-476d-8ddf-110763d7adfb.png)